react/src/components/toggle/SprkToggle.test.js

```diff
@@ -165,7 +165,10 @@ describe('SprkToggle:', () => {
   it(`should add classes to the title if triggerTextAdditionalClasses
   are set`, () => {
     const wrapper = mount(
-      <SprkToggle triggerText="Toggle title" triggerTextAdditionalClasses="test">
+      <SprkToggle
+        triggerText="Toggle title"
+        triggerTextAdditionalClasses="test"
+      >
         Body text
       </SprkToggle>,
     );
```

Comment:
Same feedback as Masthead - let's make sure to add a new test to verify that the ID doesn't change on toggle.